src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/trust/TrustToReviewDao.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dao.trust;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class TrustToReviewDao extends TrustDataDao {
```

Comment:
Rename TrustDataToReviewDao to make clear this is an extension of TrustDataDao

Reply:
Done :thumbsup: